examples/xcm/asset_teleportation.eg.ts

```diff
@@ -39,22 +42,30 @@ console.log("Alice initial free:", aliceFreeInitial)
 
 XcmPallet
   .limitedTeleportAssets({
-    dest: VersionedMultiLocation.V2(Rune.rec({
-      parents: 0,
-      interior: XcmV2Junctions.X1(XcmV2Junction.Parachain(1000)),
-    })),
-    beneficiary: VersionedMultiLocation.V2(Rune.rec({
-      parents: 0,
-      interior: XcmV2Junctions.X1(XcmV2Junction.AccountId32({
-        id: alice.publicKey,
-        network: XcmV2NetworkId.Any(),
-      })),
-    })),
-    assets: VersionedMultiAssets.V2(Rune.array([Rune.rec({
-      id: XcmV2AssetId.Concrete(Rune.rec({
+    dest: VersionedMultiLocation.V2(
+      XcmV2MultiLocation({
         parents: 0,
-        interior: XcmV2Junctions.Here(),
-      })),
+        interior: XcmV2Junctions.X1(XcmV2Junction.Parachain(1000)),
+      }),
+    ),
+    beneficiary: VersionedMultiLocation.V2(
+      XcmV2MultiLocation({
+        parents: 0,
+        interior: XcmV2Junctions.X1(
+          XcmV2Junction.AccountId32({
+            id: alice.publicKey,
+            network: XcmV2NetworkId.Any(),
+          }),
+        ),
+      }),
+    ),
+    assets: VersionedMultiAssets.V2(XcmV2MultiAssets([XcmV2MultiAsset({
```

Comment:
I'm back and forth regarding `XcmV2MultiAssets` vs. `Rune.array`. Given that the signature of `VersionedMultiAssets.V2` reflects a parameter type of `XcmV2MultiAssets`, it may be more intuitive.